src/main/java/net/atos/zac/app/admin/converter/RESTCaseDefinitionConverter.java

```diff
@@ -31,15 +31,17 @@ public class RESTCaseDefinitionConverter {
     @Inject
     private RESTHumanTaskReferentieTabelConverter restHumanTaskReferentieTabelConverter;
 
-    public RESTCaseDefinition convertToRESTCaseDefinition(final String caseDefinitionKey) {
+    public RESTCaseDefinition convertToRESTCaseDefinition(final String caseDefinitionKey, boolean inclusiefRelaties) {
```

Comment:
even een final voor boolean inclusiefRelaties

Reply:
neem ik mee in de volgende commit